src/main/java/com/cinema/config/AppConfig.java

```diff
@@ -14,7 +14,8 @@
 @PropertySource("classpath:db.properties")
 @ComponentScan(basePackages = {
         "com.cinema.dao",
-        "com.cinema.service"
+        "com.cinema.service",
+        "com.cinema.model.dto"
```

Comment:
Is that necessary?
Take a look at your getSessionFactory method

Reply:
Isn't necessary, TY